From: =?utf-8?q?Rapha=C3=ABl_Hertzog?= <hertzog@debian.org>
Date: Tue, 21 Jul 2015 16:06:57 +0200
Subject: Avoid call to git ls-files via gemspec

The call to git ls-files generates spurious error messages
on a package setup that does not have any .git metadata.
---
 metasploit-framework.gemspec | 3 ---
 1 file changed, 3 deletions(-)

--- a/metasploit-framework.gemspec
+++ b/metasploit-framework.gemspec
@@ -24,9 +24,6 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://www.metasploit.com'
   spec.license       = 'BSD-3-clause'
 
-  spec.files         = `git ls-files`.split($/).reject { |file|
-    file =~ /^documentation|^data\/gui|^external/
-  }
   spec.bindir = '.'
   if ENV['CREATE_BINSTUBS']
     spec.executables   = [
